src/pages/HomePage.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import '../App.css'
+import Landing from '../components/Landing'
+import About from '../components/About'
+import Why from '../components/Why'
+import Register from '../components/Register'
+import FAQ from '../components/FAQ'
+import Schedule from '../components/Schedule'
+import Sponsors from '../components/Sponsors'
+import Contact from '../components/Contact'
+
+export default function Home () {
```

Comment:
I dont see point of this home page component. Like stated earlier about no need for router this can all be in the app.js